perun-base/src/main/resources/perun-roles.yml

```
@@ -8181,13 +8186,33 @@ perun_roles_management:
       - PERUNADMIN:
     privileged_roles_to_read:
       - PERUNADMIN:
+      - PERUNADMINBA:
       - PERUNOBSERVER:
     associated_read_roles:
       - PERUNOBSERVER
     assignable_to_attributes: false
     mfa_critical_role: true
     display_name: "Perun admin"
 
+  PERUNADMINBA:
+    primary_object:
+    assign_to_objects: {}
+    assignment_check:
+      - MFA:
+    entities_to_manage:
+      User: user_id
+    privileged_roles_to_manage:
+      - PERUNADMINBA:
+    privileged_roles_to_read:
+      - PERUNADMIN:
+      - PERUNADMINBA:
+      - PERUNOBSERVER:
+    associated_read_roles:
+      - PERUNOBSERVER
+    assignable_to_attributes: false
+    mfa_critical_role: false
+    display_name: "Perun admin BA"
```

```suggestion
    display_name: "Perun admin BA"
    skip_mfa: true
```
this is missing in order to skip MFA when performing critical actions